wiki/People/The_Team/Nomination_Assessment_Team/SEV_rating/en.md

```diff
@@ -0,0 +1,161 @@
+| 1 | Worth noting | Impacts gameplay negatively but not significantly. |
+| 2 | Moderate design flaw | Harms gameplay to the point where it's typically also noticeable to a regular user, e.g. a big jump in a low difficulty. In practice, this is often a combination of multiple notable reasons, e.g. being both too difficult to read and an unwarranted difficulty spike. |
+
+Below are examples of SEV ratings and how they are roughly interpreted by evaluators:
+
+| SEV | Description |
+| :-- | :-- |
+| **0/0** | This reset is insignificant and ignored for the purpose of evaluations. |
+| **0/1** | Something went wrong here but it’s hard to blame the BNs, as it was difficult to spot. |
+| **1/0** | Could've been fixed if the BNs had been more attentive, but isn’t a significant issue. |
+| **1/1** | Something went wrong here that could have been fixed if the BNs were more attentive. |
+| **1/2** | Often means many things went wrong, but all require experience to spot easily. |
+| **2/0** | Some glaring issue in the map’s settings, like metadata, was somehow missed. |
+| **2/1** | Some glaring issue in the map's gameplay, like no hitsounding, was missed. |
+| **2/2** | An issue so severe that it is difficult not to spot, like concurrent hit objects, was missed. |
+
+## How SEV is used
+
+SEV ratings are used in the evaluations of the Beatmap Nominators, weighed against how many nominations they've done.
+
+Mistakes happen and a healthy amount will help with learning, but if they happen too frequently or the same mistakes repeat, then that's a problem. This is why evaluations don't focus on individual SEV ratings, but take the entire picture into account on a case-by-case basis.
+
+## Common reset reasons
+
+*Makes up 90% of resets.*
+
+Here is an exhaustive list of reasons nominations have been reset for with their respective SEV rating. These are based on stats of all SEV ratings set in the osu! game mode from February 2020 to April 2021, with percentages to show how common issues are.
+
+***Important to note:*** *This list does not cover every possible reset reason, and the NAT may rate resets for the same reasons listed differently, depending on the specific context.*
+
+### Metadata
+
+*Makes up 22% of notable resets, 30% of resets.*
+
+Metadata resets *never* have a severity above 0, as they don't affect gameplay.
+
+- **0/0:** (70%)
+  - Adding tags for a new Featured Artist
+  - Adding guest mappers to tags due to a username change
+  - Adding tags that are more descriptive but aren’t required by the Ranking Criteria
+  - Resets due to the addition of a new rule
+  - Difficulty name changes
+- **1/0:** (23%)
+  - Romanised artist name order
+  - Small romanisation and capitalisation errors
+  - 1 wrong character, spelling mistakes
+  - Missing genre/language tags
+- **2/0:** (5%)
+  - Missing guest mappers from tags
+  - Missing unicode field
+  - Wrong artist/title/source
+
+### Mapping
+
+*Makes up 23% of notable resets, 18% of resets.*
```

Comment:
either:

- use "regular resets" for all of these, to make it clear that they're not supersets of "notable resets"
- use "of all resets", and sum up the values (23+18=41%)

correct me if i'm misunderstanding this

Reply:
This terminology ("notable resets" = "resets" that are not 0/0) is likely only understood by NAT, so changing or clarifying this is probably a good idea.

However, the difference between *any* resets and *notable* resets is massive, so let's not sum it up. The intent behind the percentages is to show how often they are considered in evaluations compared to other reset categories. Through this we can, for example, see that metadata resets are considered much less than timing resets.

This would be less vague and a step in the right direction if nothing else:
```suggestion
*Makes up 23% of all >0/0 resets, 18% of all resets.*
```

Reply:
ah ok, let's go with that wording then